### PR TITLE
Update AzureDevOpsPushPipelineExtension.md

### PR DESCRIPTION
**Pull request by zhenlan**

### CODE REVIEW

**Comment by zhenlan** [comment]
@Matthew-Solomon

File: releaseNotes/AzureDevOpsPushPipelineExtension.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Azure App Configuration Push Extension release notes
-Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version of the Azure App Configuration task will print to the console when the task is executed: ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
+Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information will print to the console when the task is executed:
+![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
```
Comment (abhilasharora) [inline releaseNotes/AzureDevOpsPushPipelineExtension.md:3]: >pictures/AzureDevOpsPushExtensionVersionSample.PNG [](start = 10, length = 50)

Should we update the extension and image to show the preview tag in the version?

File: releaseNotes/AzureDevOpsPushPipelineExtension.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ## Azure App Configuration Push Extension release notes
-Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version of the Azure App Configuration task will print to the console when the task is executed: ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
+Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information will print to the console when the task is executed:
```
Comment (abhilasharora) [inline releaseNotes/AzureDevOpsPushPipelineExtension.md:2]: >will print [](start = 252, length = 10)

`will print` => `is printed` OR `will be printed`

**Comment by abhilasharora** [comment]
> @Matthew-Solomon

@Matthew-Solomon Can you join the following team so we can add you as a collaborator in GitHub PRs?
https://repos.opensource.microsoft.com/Azure/teams/azure-appconfig-team

File: releaseNotes/AzureDevOpsPushPipelineExtension.md (hunk 1 above)
Comment (zhenlan) [reply releaseNotes/AzureDevOpsPushPipelineExtension.md:3]: The "preview" tag is actually not part of the version on the ADO marketplace page. This is for illustration anyway. We probably won't update the image even when we release newer versions in the future.

File: releaseNotes/AzureDevOpsPushPipelineExtension.md (hunk 2 above)
Comment (zhenlan) [reply releaseNotes/AzureDevOpsPushPipelineExtension.md]: Updated.

**Pull request merged**